src/components/ListItem.js

```diff
@@ -0,0 +1,27 @@
+import { useProducts } from '../hooks/useProducts';
+
+const ListItem = () => {
+  const { products, loading, error } = useProducts();
+
+  const showProducts = () => {
+    if (products.length === 0) {
+      return <p>You don't have any product in your list</p>;
+    }
+
+    return products.map(({ id, name, date }) => (
+      <li key={id}>{`Product: ${name} => Created at: ${new Date(
+        date,
+      ).toLocaleString()}`}</li>
+    ));
+  };
+
+  return (
+    <div>
+      {loading ? <p>Loading...</p> : null}
+      {error ? <p>There was an error</p> : null}
+      {!loading ? showProducts() : null}
```

Comment:
What do you think about simplifying this logic a bit so we only have to handle two conditions?

```suggestion
      {loading ? <p>Loading...</p> : showProducts()}
      {error ? <p>There was an error</p> : null}
```